frontend/src/tests/pages/LeaderboardPage.test.js

```diff
@@ -1,36 +1,97 @@
-import { render, screen } from "@testing-library/react";
+import { _fireEvent, render, _screen, _waitFor } from "@testing-library/react";
+import _mockConsole from "jest-mock-console";
```

Comment:
I'd prefer if the unused imports were removed entirely instead of being prefixed with an underscore.

```suggestion
import { render } from "@testing-library/react";
```